tests/testthat/test-report_constructors.R

```diff
@@ -0,0 +1,222 @@
+context("Report constructor")
+local_edition(3)
+
+library(htmltools)
+
+results_test <- function() {
```

Comment:
Looks suspiciously similar to `report_test` function. Would it benefit from using it? I mean something like: 
```
return(get_results(report_test()))
```
It might require loading the functions together, maybe as a part of `testthat.R`

Reply:
function updated in [880768c](https://github.com/Appsilon/data.validator/pull/63/commits/880768c014f4fd31f628bec633164642acec2a68)